### PR TITLE
examples

### PR DESCRIPTION
**Pull request by jburel**
Adjusts the remaining python scripts under examples
We can check what we want to migrate/rescue from the examples folder later on

### CODE REVIEW

File: examples/OmeroClients/collectionmethods.py [hunk 1]
```diff
@@ -8,14 +8,14 @@
 EventI = omero.model.EventI
 PixelsI = omero.model.PixelsI
 
-image = ImageI(long(1), True)
-image.getDetails().setUpdateEvent(EventI(1L, False))
+image = ImageI(int(1), True)
```
Comment (joshmoore) [inline examples/OmeroClients/collectionmethods.py]: No objection, but the `int` here is almost certainly no longer needed.

**Review by joshmoore — APPROVED** [review]
Changes all look fine. :+1:

File: examples/OmeroClients/collectionmethods.py (hunk 1 above)
Comment (jburel) [reply examples/OmeroClients/collectionmethods.py]: removed

**Comment by jburel** [comment]
Changes approved. Merging

**Pull request merged**